### PR TITLE
Tweaked protocol viewer UI

### PR DESCRIPTION
**Pull request by bvaughn**
https://www.loom.com/share/e31c583a16fe48d0a90f672058111aa8

### CODE REVIEW

File: src/ui/components/ProtocolViewer.tsx [hunk 1]
```diff
@@ -123,38 +131,54 @@ const ProtocolViewer = () => {
...
-          >
-            <span>
-              {chunk.method}
-              {chunk.count > 1 ? `(${chunk.count})` : null}
-            </span>
-            <span>{msAsMinutes(chunk.startedAt)}</span>
-          </div>
-        );
-      })}
+    <div className={styles.Container}>
```
Comment (jcmorrow) [inline src/ui/components/ProtocolViewer.tsx:134]: My only comment is: this could be a good place for an accordion 🤷‍♂️

**Review by jcmorrow — APPROVED** [review]

File: src/ui/components/ProtocolViewer.tsx (hunk 1 above)
Comment (bvaughn) [reply src/ui/components/ProtocolViewer.tsx:134]: Oh! That is a fantastic idea.

Comment (bvaughn) [reply src/ui/components/ProtocolViewer.tsx:134]: We seem to have at least **four** accordion components in the devtools repo 😁  

The first two I've looked at don't support dynamic children (e.g. rendering with only the requests pane at first and adding details panes later). They will throw if you do that. So they wouldn't work very well for this case.

I think I'll leave that out for now 😁

**Comment by bvaughn** [comment]
Btw @jcmorrow– as I look at the `chunkedRequests` method again– I'm wondering what's the purpose of the default chunk we push into the accumulated array? Given that its' method is "" – do we expect any protocol requests to match it? I would assume we _don't_ and we'll just end up with an empty chunk at the top of every array.

I'm going to try removing it (and updating the reducer to account for that).

**Pull request merged**